hazelcast/src/main/java/com/hazelcast/internal/memory/impl/UnsafeUtil.java

```diff
@@ -55,7 +51,6 @@
             }
         } catch (Throwable t) {
             unsafe = null;
-            LOGGER.warning("Unable to get an instance of Unsafe. Unsafe-based operations will be unavailable", t);
```

Comment:
I also see this as a problematic. Unsafe may be unavailable due security manager / jigsaw / difference JVM and you wants to know this.